web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx

```diff
@@ -33,10 +34,10 @@ function StorePlugin(props: StorePluginProps) {
   const params = { source: url };
 
   useEffect(() => {
-    if (isInstallSuccess) {
+    if (isInstallSuccess && !isInstallLoading) {
```

Comment:
are you sure we need that ? isn't isInstallLoading always false if isInstallSuccess = true ?